pygmt/src/grdtrack.py

```diff
@@ -9,22 +9,44 @@
     build_arg_string,
     data_kind,
     fmt_docstring,
+    kwargs_to_strings,
     use_alias,
 )
 
 
 @fmt_docstring
-@use_alias(V="verbose", f="coltypes", n="interpolation")
+@use_alias(
+    A="resampling",
```

Comment:
Would you be open to `resample` rather than `resampling`? I think the "ing"-form alias is out of place relative to the form of other aliases in PyGMT.
```suggestion
    A="resample",
```

Reply:
Yep, sounds reasonable, done in c10eba6d79d4b622fe01f0a5e330533e21a2afdf.